File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestion_buses/bus_envio/bBusEnvioDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestion_buses.bus_envio;

import lombok.*;
import java.io.Serializable;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class bBusEnvioDTO implements Serializable {
    UUID id;
    String placa;
    String modelo;
    List<bRutaEnvioDTO> rutas;
}

File: frontend-angular/src/app/dto/gestionar-bus/bus-envio/bBusEnvioDTO.ts
import {BRutaEnvioDTO} from './bRutaEnvioDTO';

export class BBusEnvioDTO {
  constructor(
    public id: string,
    public placa: string,
    public modelo: string,
    public diasSemana: string[],
    public rutas: BRutaEnvioDTO[]
  ) {}
}
